packages/react/src/primitives/CheckboxField/CheckboxField.tsx

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { Checkbox } from '../Checkbox';
+import { FieldErrorMessage, FieldDescription } from '../Field';
+import { Flex } from '../Flex';
+import { CheckboxFieldProps } from '../types/checkboxField';
+import { ComponentClassNames } from '../shared';
+
+export const CheckboxField: React.FC<CheckboxFieldProps> = ({
+  alignContent,
+  alignItems,
+  checked,
+  children,
+  className,
+  defaultChecked,
+  descriptiveText,
+  direction = 'column',
+  errorMessage,
+  gap,
+  hasError = false,
+  id,
+  isEmphasized,
+  isDisabled,
+  isReadOnly,
+  isRequired,
+  justifyContent,
+  label,
+  labelHidden = false,
+  name,
+  onChange,
+  testId,
+  size,
+  value,
+  wrap,
+  ...rest
+}) => {
+  return (
+    <Flex
+      alignContent={alignContent}
```

Comment:
Most of these Flex style properties are not going to have any affect on the primary layout of the `Checkbox` component, which seems confusing. What do you think of passing the Flex style props down to the Checkbox component instead? That seems like a more common use case for the flex style props. If customers really need to change the position of the Error Message, they could use CSS. What do you think?

Reply:
Yea, that make sense